packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -228,6 +229,18 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to null. If null, only the text field is updated.
   final ValueChanged<T?>? onSelected;
 
+  /// Determine if the dropdown button requests focus and the on-screen virtual
+  /// keyboard is shown in response to a touch event.
+  ///
+  /// By default, if the app is running on mobile platforms, the text field
```

Comment:
By default, on mobile platforms, tapping on the text field and opening the menu will not case a focus request and the virtual keyboard will not appear. The default behavior for desktop platforms is for the dropdown to take the focus.